index.html

```diff
@@ -38,6 +38,7 @@
     <h1>Livro de Receitas do cesupa</h1>
     <ul>
         <li><a href="lasanha-a-bolonhesa.html">Lasanha à Bolonhesa</a></li>
+        <li><a href="bolo.html">Lasanha à Bolonhesa</a></li>
```

Comment:
Altere para o nome da sua receita

```suggestion
        <li><a href="bolo.html">Bolo de Cenoura</a></li>
```